pkg/api/v1alpha2/api.proto

```diff
@@ -378,8 +378,8 @@ message TrialStatus {
 
 message Trial {
     string name = 1;
-    TrialSpec spec = 2;
-    TrialStatus status = 3;
+    TrialSpec trial_spec = 2;
```

Comment:
In fact, I like spec/status rather than trial_spec/trial_status. Maybe we should refactor experiment_spec/experiment_status instead of here

Reply:
I can rename experiment_spec to spec, if we want to. 
What do you think? @johnugeorge @gaocegege @richardsliu

Reply:
Prefer spec/status but it may require lots of work to update the code base.

Reply:
I will update it in this PR, no problem.

Reply:
Thanks